docs/Select-Canonical-Transcript.md

```diff
@@ -0,0 +1,92 @@
+# How Genome Nexus selects a canonical transcript
+
+## **Background**
+When a variant is sent to Genome Nexus, Genome Nexus calls **VEP** (Variant Effect Predictor)to get a range of annotations associated with the variant, including a list of potential transcripts.
+
+Genome Nexus then uses a set of predefined criteria to select the canonical transcript from this list of transcripts. The selection process is guided by the user-selected isoform override source and a prioritization logic.
+
+The canonical transcript is in the `transcriptConsequenceSummary` field, under the `annotation_summary` section of the response. To ensure the canonical transcript is returned, the `annotation_summary` field must be requested in the query sent to Genome Nexus.
+
+Example response structure:
+```
+"annotation_summary" {
+    "transcriptConsequenceSummary" {
+        ...
+    }
+}
+```
```

Comment:
this is quite technical, I'm wondering if it's better to split this out in separate sections, e.g. "command-line client", "REST API", "website" etc. The basics of the prioritization logic is useful for all users, but some details are only relevant for a subset of downstream users